### PR TITLE
[feat] Reorder and add non-hideable table columns

### PR DESCRIPTION
**Pull request by arsengit**
- Added `RunNameColumn` component for rendering showing status indicator and run name
- Added `GroupHeading` component for making grouping headers more distinguishable
- Fixed manage columns functionality in images explorer
- Added `getColumnOptions` util function
- Added non-hideable table columns to table config
- Reordered columns default order
- Renamed heading column names

### CODE REVIEW

**Comment by arsengit** [comment]
Resolves #1806

File: aim/web/ui/src/utils/getColumnOptions.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import { IOnGroupingSelectChangeParams } from 'types/services/models/metrics/metricsAppModel';
+
+const icons: { [key: string]: string } = {
+  color: 'coloring',
+  stroke: 'line-style',
+  chart: 'chart-group',
+  group: 'image-group',
+};
+
+const groupList: any = {
```
Comment (KaroMourad) [inline aim/web/ui/src/utils/getColumnOptions.ts:12]: let's use here `AppNameEnum` as object keys and add the correct type here

File: aim/web/ui/src/components/CustomTable/Table.scss [hunk 2]
```diff
@@ -74,7 +74,37 @@ $border-radius: 0.2rem;
...
+  height: 0.75rem;
+  padding: 0;
+  background-color: unset !important;
+  position: absolute;
+  left: 1px;
+  top: calc(50% - 6px);
+  &::before {
+    top: toRem(2px);
+    left: toRem(2px);
+    // animation: indicator-pulse 2s infinite;
```
Comment (VkoHov) [inline aim/web/ui/src/components/CustomTable/Table.scss:88]: let's remove all redundant comments

**Review by VkoHov — APPROVED** [review]
lgtm!

**Review by KaroMourad — APPROVED** [review]
lgtm

**Pull request merged**